packages/office-addin-lint/src/lint.ts

```diff
@@ -32,8 +32,12 @@ export function performLintCheck(files: string) {
     execCommand(command);
     usageDataObject.sendUsageDataSuccessEvent("performLintCheck");
   } catch (err) {
-    usageDataObject.sendUsageDataException("performLintCheck", err);
-    throw err;
+    if (err.status && err.status == ESLintExitCode.LintingError) {
```

Comment:
why are we sending linting errors to usage data. Perhaps I am misunderstanding, but we don't really care about users' linting error and don't want to muddy up our own usage data with this type of data.  If I am misunderstanding, it's because there's no description for this PR. Please be sure to include this for a PR :-)

Reply:
We want to report a successful command execution when there is a non-zero exit code (which results in an exception being thrown), and report the exit code, but report a failure when the status code indicates the command failed (which is a value of 2).